mftf/2.2/test/action-groups.md

```diff
@@ -27,7 +27,7 @@ The XML format for the `actionGroups` declaration is:
 <?xml version="1.0" encoding="UTF-8"?>
 
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:noNamespaceSchemaLocation="../../../../../../dev/tests/acceptance/vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/testSchema.xsd">
+        xsi:noNamespaceSchemaLocation="../../../../../../dev/tests/acceptance/vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/actionGroupSchema.xsd">
```

Comment:
I did not intend on changing these URNs because `MFTF 2.2.0` did not have a command for generating the URNs, or upgrading test materials to have these URNs.

As far as I know in this codebase `mftf/2.2` means the MFTF version, not magento version. If that assumption is incorrect I agree with the suggested change and will commit to it.